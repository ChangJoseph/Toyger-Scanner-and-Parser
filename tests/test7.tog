let
  var x: int

  function f(x:int) :void = 
    printint(x+20);
    printstring("Done")
  end

  var y := 2+3

  function g(y:int): int = 
    return y - 1
  end

in
  x := getint();
  f(g(x + y))
end
//end of the program
