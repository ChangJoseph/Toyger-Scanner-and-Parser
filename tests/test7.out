Global Variable x: line 2
Global Variable y: line 9

Number of functions defined: 2
Number of comments: 1
Number of strings: 1
